src/flows/shared/steps/OrganizeCollection/Editor/SortableStagedBlankBlock.tsx

```diff
@@ -0,0 +1,125 @@
+import { useSortable } from '@dnd-kit/sortable';
+import colors from 'components/core/colors';
+import { BodyRegular, Caption } from 'components/core/Text/Text';
+import transitions from 'components/core/transitions';
+import { useEffect, useMemo, useRef } from 'react';
+import styled, { css, keyframes } from 'styled-components';
+import { CSS } from '@dnd-kit/utilities';
+import UnstageButton from './UnstageButton';
+
+type Props = {
+  id: string;
+  size: number;
+};
+
+export default function SortableStagedBlankBlock({ id, size }: Props) {
+  const { attributes, listeners, isDragging, setNodeRef, transform, transition } = useSortable({
```

Comment:
lines 16-27 to get the dnd related props is duplicated across this file and `SortableStagedNft`, but I decided it was ok because it's a pain to prop drill these from a shared component, and it doesn't make sense to use a context because it's just two files and the properties are unique for every item in the dnd.

Reply:
we can revisit if we add another kind of sortable staged item in the future